sigstore-java/build.gradle.kts

```
@@ -120,3 +121,7 @@ jsonSchema2Pojo {
 tasks.named("sourcesJar") {
     dependsOn("generateJsonSchema2DataClassConfigRekor")
 }
+
+tasks.getByName("generateBuildInfo") {
+    (this as buildlogic.BuildInfoTask).packageName.set("dev.sigstore.buildinfo")
```

Does the following work?

```suggestion
tasks.generateBuildInfo {
    packageName.set("dev.sigstore.buildinfo")
```